src/core/util/util.js

```diff
@@ -57,12 +57,27 @@ function UtilFactory($document, $timeout, $compile, $rootScope, $$mdAnimate, $in
 
   }
 
+  /**
+   * Bulletproof method to get a function to return the current timestamp
+   * @returns {Function}
+   */
+  function getNowMethod() {
+    if (window.performance && typeof window.performance.now === 'function') {
+      return angular.bind(window.performance, window.performance.now);
```

Comment:
We should be able to use the `Function.bind` method, since it supports all browsers we support.
> See https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Function/bind

```js
return window.performance.now.bind(window.performance)
```